specifications/api/swagger.yaml

```diff
@@ -2755,6 +2858,47 @@ components:
             - <<: *transactionWithdrawalRequestSelf
             - <<: *transactionRedemptionRequest
 
+    ApiTransactionExtended: &ApiTransactionExtended
+      type: object
+      required:
+        - cborHex
+        - description
+        - type
+      properties:
+        cborHex: *serialisedTransactionHex
+        description: *txDescription
+        type: *txType
+
+    ApiExternalInput: &ApiExternalInput
+      type: object
+      required:
+        - txIn
+        - txOut
+      properties:
+        txIn: *txInNode
+        txOut: *txOutNode
+
+    ApiExternalInputs: &ApiExternalInputs
+      description: A flat list of node compliant inputs.
```

Comment:
I think I'm missing something, can't we just get the inputs from the serialised transaction itself?